src/Hedgehog/Report.fs

```diff
@@ -29,6 +35,45 @@ type Report = {
     Status : Status
 }
 
+
+module internal RecheckData =
+    open System
+
+    module Base64 =
+        open System.Text
+
+        let encode (plainText: string) =
+            plainText
+            |> Encoding.UTF8.GetBytes
+            |> Convert.ToBase64String
+
+        let decode (cipherText: string) =
+            cipherText
+            |> Convert.FromBase64String
+            |> Encoding.UTF8.GetString
+
+    let private separator = "\n"
+
+    let serialize data =
+        data.Size.ToString ()         + separator
+        + data.Seed.Value.ToString () + separator
+        + data.Seed.Gamma.ToString ()
+        |> Base64.encode
```

Comment:
```fsharp
[ string data.Size
  string data.Seed.Value
  string data.Seed.Gamma ]
|> String.concat separator
|> Base64.encode
```

Maybe?

Reply:
Sure.  Just include this change in a force push.